evidence/pool_test.go

```diff
@@ -314,8 +387,9 @@ func initializeValidatorState(valAddr []byte, height int64) dbm.DB {
 				MaxGas:   -1,
 			},
 			Evidence: tmproto.EvidenceParams{
```

Comment:
does maxNum need to bee here?

Reply:
yup it should be - I thought it was already there